frontend/src/templates/list/ListTemplate.tsx

```diff
@@ -84,6 +89,9 @@ export default function ListTemplate<T extends IDer>(
   const onPage = (event: DataTablePageEvent) => {
     logger.debug("Page Applied", event);
     setNumCurrentlyDisplayedRows(event.rows);
+    event.rows === props.totalNumberOfEntries
```

Comment:
Just wondering if anything funky would happen if the number of rows happens to equal the number of entries, but the user has not selected no pagination. Like if there are 10 books, and the user selects to show 10 per page. I think nothing would happen, but just want to make sure